# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/browser/browser_nsIFormPOSTActionChannel.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/browser/browser_nsIFormPOSTActionChannel.js
--- a/netwerk/test/browser/browser_nsIFormPOSTActionChannel.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/browser/browser_nsIFormPOSTActionChannel.js	Tue Feb 12 16:08:25 2019 +0000
@@ -115,10 +115,7 @@
   open: function() {
     throw Cr.NS_ERROR_NOT_IMPLEMENTED;
   },
-  open2: function() {
-    throw Cr.NS_ERROR_NOT_IMPLEMENTED;
-  },
-  asyncOpen: function(aListener, aContext) {
+  asyncOpen: function(aListener) {
     var data = `
 <!DOCTYPE html>
 <html>
@@ -179,21 +176,18 @@
     var runnable = {
       run: () => {
         try {
-          aListener.onStartRequest(this, aContext);
+          aListener.onStartRequest(this, null);
         } catch(e) {}
         try {
-          aListener.onDataAvailable(this, aContext, stream, 0, stream.available());
+          aListener.onDataAvailable(this, null, stream, 0, stream.available());
         } catch(e) {}
         try {
-          aListener.onStopRequest(this, aContext, Cr.NS_OK);
+          aListener.onStopRequest(this, null, Cr.NS_OK);
         } catch(e) {}
       }
     };
     Services.tm.dispatchToMainThread(runnable);
   },
-  asyncOpen2: function(aListener) {
-    this.asyncOpen(aListener, null);
-  },
 
   /** nsIRequest */
   get name() {